.travis.yml

```diff
@@ -15,3 +15,4 @@ script:
   - git-validation -run DCO,short-subject -v -range ${TRAVIS_COMMIT_RANGE}
   - make
   - make test
+  - PATH="${PATH}:${PWD}" make -C test
```

Comment:
I think this will be failed.  As from test/README.md, we need do 'git submodule update --init' first. But you didn't do such preparation in travis.yml